harvester/vm-import-controller
# vm-import-controller enhancements

PR introduces changes to vm import controller to improve the following:
* change behaviour of vm-import-controller to disable retry for failed vm-import-controller requests if export VM fails. This is to avoid triggering multiple exports on source environment
* allow definition of custom storage classes when specifying virtualmachineimports
* detect source virtual machine disk bus types and try and use similar bus on harvester
* detect firmware settings and accordingly setup bios/uefi/secure boot/tpm in harvester
* update govmomi to latest version
* update gophercloud to latest version
* update openstack snapshot readiness interval to 2hrs, as this can take longer for larger images
* change how vmware client closes the export lease to ensure the export does not timeout

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**


**Solution:**


**Related Issue:**
https://github.com/harvester/harvester/issues/5758

**Test plan:**
